db/seeds.rb

```diff
@@ -40,15 +40,27 @@
     end
   end
 
+  {
+    'Contact Centre Manager' => 'manager',
```

Comment:
The roles need to be available in production/staging etc.  I don't think we should run the seeds.rb file in production.

Reply:
Yep, definitely, there's a need for a data migration scripts to create and assign these roles, will add a trello card for that